package/MDAnalysis/coordinates/PQR.py

```diff
@@ -128,7 +128,7 @@ def _read_first_frame(self):
             for line in pqrfile:
                 if line.startswith(('ATOM', 'HETATM')):
                     fields = line.split()
-                    coords.append(map(float, fields[-5:-2]))
+                    coords.append(np.float32(fields[-5:-2]))
```

Comment:
So this is me being really pedantic, (but loops like this scale with number of particles, so it's important), but making lists is cheaper than making arrays, so I'd rather have list comprehensions here